dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -36,7 +36,19 @@ cmd_config() {
     docker_run -v "${CHE_HOST_CONFIG}":/copy \
                -v "${CHE_HOST_DEVELOPMENT_REPO}"/dockerfiles/init:/files \
                   $IMAGE_INIT
+  fi
+
+  info "config" "Generating $CHE_MINI_PRODUCT_NAME configuration..."
+  # Run the docker configurator
+  generate_configuration_with_puppet
+
+  # Replace certain environment file lines with their container counterparts
+  info "config" "Customizing docker-compose for running in a container"
+
+  # Write the installed version to the *.ver file into the instance folder
+  echo "$CHE_VERSION" > "${CHE_CONTAINER_INSTANCE}/${CHE_VERSION_FILE}"
```

Comment:
I've moved that line within `cmd_config`. I don;t understand your concern because that line is  already in `cmd_config` I've just moved it before dev mode things which is more accurate and more consistent.